seeders/20231010142838-seedTodo.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    /**
+     * Add seed commands here.
+     *
+     * Example:
+     * await queryInterface.bulkInsert('People', [{
+     *   name: 'John Doe',
+     *   isBetaMember: false
+     * }], {});
+     */
+
+    await queryInterface.bulkInsert(
+      "Todos",
+      [
+        {
+          title: "Create judul todos satu",
+          description: "description todos satu",
+          status: "created",
+          createdAt: new Date(),
+          updatedAt: new Date(),
+        },
+        {
+          title: "Create judul todos dua",
+          description: "description todos dua",
+          status: "created",
+          createdAt: new Date(),
+          updatedAt: new Date(),
+        },
+        {
+          title: "Create judul todos tiga",
+          description: "description todos tiga",
+          status: "created",
+          createdAt: new Date(),
+          updatedAt: new Date(),
+        },
+      ],
+      {}
+    );
+  },
```

Comment:
seednya minta tolong tambahin lagi aja yaa 2, 
sisanya goodjob
structuring folder udah cakep router sama controller udah dipake
erd cakep
express jalan semua
migration aman
seed kurang dikit
hooks and validation masuk. goodjob!